pinns/src/pinns.c

```diff
@@ -254,6 +263,11 @@ int main(int argc, char **argv) {
   return EXIT_SUCCESS;
 }
 
+// returns true if the option is empty or not equal to 'host'
+static bool is_not_host_ns(const char* const optarg) {
```

Comment:
I think inverting this function to `is_host_ns` and adding the `!` at the caller is easier to read.

Reply:
to be honest I forget why I did this...